.github/workflows/test.yml

```diff
@@ -126,7 +126,7 @@ jobs:
     name: Visual Regression Tests
     runs-on: ubuntu-latest
     container:
-      image: mcr.microsoft.com/playwright:v1.39.0-jammy
+      image: mcr.microsoft.com/playwright:v1.44.1-jammy
```

Comment:
We should have also updated `.github/workflows/update-visual-tests.yml`. PR here: https://github.com/kiwicom/orbit/pull/4373